aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -50,26 +50,42 @@ In this tutorial, you learn how to:
 
 Configure Visual Studio to look for npm in the *PATH* environment variable. By default, Visual Studio uses the version of npm found in its installation directory. Follow these instructions in Visual Studio:
 
+1. Launch Visual Studio 2019. At the start window, select **Continue without code**.
```

Comment:
```suggestion
1. Launch Visual Studio. At the start window, select **Continue without code**.
```

VS 2019 is listed as a prereq above. For that reason, let's drop the version number here.

Reply:
One less thing to update in the next version change. We never specify the version in the topic outside the prereq .